### PR TITLE
Make SVG icons into an enum rather than const. Also add /dev/icons view to determine the size of each icon.

### PR DESCRIPTION
**Pull request by clouser2**
### Description

Moving this to an enum makes it easier to categorize what icons we actually have. Additionally, I've been dealing with issues where I try and scale an existing SVG path up and down and running into issues where the content gets clipped. This is due to the fact that the SVG paths we have have inherent sizes. We'll need to set the viewBox property to this inherent size in order to properly scale the icon up/down using the width/height properties. Having the `/dev/icons` view is a first step to determining which icons need to be updated (and is also generally useful for seeing what icons are available).

See https://css-tricks.com/scale-svg/.

### CODE REVIEW

File: server/app/assets/javascripts/dev_icons.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+window.addEventListener('load', () => {
+  Array.from(document.querySelectorAll('tr')).forEach((rowEl) => {
```
Comment (bion) [inline server/app/assets/javascripts/dev_icons.ts:2]: suggestion (blocking): please document this file

File: server/app/views/dev/IconsView.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import com.google.inject.Inject;
+import j2html.tags.ContainerTag;
+import j2html.tags.Tag;
+import play.twirl.api.Content;
+import views.BaseHtmlLayout;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.components.Icons;
+
+public final class IconsView extends BaseHtmlView {
```
Comment (bion) [inline server/app/views/dev/IconsView.java:21]: suggestion (blocking): please document this class

**Review by bion — CHANGES_REQUESTED** [review]

File: server/app/views/dev/IconsView.java (hunk 2 above)
Comment (clouser2) [reply server/app/views/dev/IconsView.java:21]: Done.

File: server/app/assets/javascripts/dev_icons.ts (hunk 1 above)
Comment (clouser2) [reply server/app/assets/javascripts/dev_icons.ts:2]: Done.

**Pull request merged**